crates/dslab-mp-python/python/dslabmp.py

```diff
@@ -97,12 +99,20 @@ def __init__(self, t: JSON):
         self.inner = t
 
     def serialize(self):
-        return json.dumps(self.inner)
+        def assertion_serializable(o):
+            assert isinstance(o, Message), 'cannot serialize custom classes' 
```

Comment:
В [документации](https://docs.python.org/3/library/json.html#json.dump) рекомендуют в этой функции выбрасывать TypeError:

If specified, default should be a function that gets called for objects that can’t otherwise be serialized. It should return a JSON encodable version of the object or raise a [TypeError](https://docs.python.org/3/library/exceptions.html#TypeError). If not specified, [TypeError](https://docs.python.org/3/library/exceptions.html#TypeError) is raised.

Reply:
И ошибку предлагаю дополнить так: 
```
cannot serialize state member using class ..., use custom implementations of get/set_state
```

Reply:
умеем со всем работать теперь, так чтоникаких TypeError не предвидится